### PR TITLE
Rename radio controls (close #422)

### LINKED ISSUE #422: Rename radio controls
Before we add `checkbox-group`, I suggest to make following changes:
- rename `radio` to `radio-group`
- rename `radio-option` to `radio`

### PR DESCRIPTION
**Pull request by dfilatov**
cc @mishanga @narqo

### CODE REVIEW

File: common.blocks/radio-group/radio-group.bemhtml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }),
+
+    content()(function() {
+        return (this.ctx.options || []).map(function(option) {
+            return {
+                block : 'radio',
+                mods : {
+                    type : this.mods.type,
+                    theme : this.mods.theme,
+                    size : this.mods.size,
```
Comment (narqo) [inline common.blocks/radio-group/radio-group.bemhtml:20]: Why don't we cache `this.mods` here?

Comment (dfilatov) [inline common.blocks/radio-group/radio-group.bemhtml:20]: This issue is about renaming only )

Comment (narqo) [inline common.blocks/radio-group/radio-group.bemhtml:20]: ok

**Comment by narqo** [comment]
:sheep: it :D

**Pull request merged**